examples/appSpec/zomes/three/capabilities/main/code/index.ts

```diff
@@ -38,6 +41,54 @@ export function test_get(encoded_allocation: u32): u32 {
 }
 
 
+/*----------  test_decode_params boilerplate  ----------*/
+
+
+var valueString: string = "";
+
+class ParameterHandler extends Handler {
+  onString(value: string): void {
+    if(this.currentKey == "keyString") {
+        valueString = value;
+    }
+  }
+}
+
+export function test_decode_params(encoded_allocation: u32): u32 {
+   let jsonString: string = deserialize(encoded_allocation);   
+
+    let handler = new ParameterHandler()
+    parseString<ParameterHandler>(jsonString, handler)
+
+   return serialize(valueString);
+
+}
+
+/*----------  test_decode_multiple_params boilerplate  ----------*/
+
+var string1: string = "";
+var string2: string = "";
+
+class MultiParameterHandler extends Handler {
+  onString(value: string): void {
+    if(this.currentKey == "firstString") {
+      string1 = value;
+    } else if (this.currentKey == "secondString") {
+      string2 = value;
+    }
+  }
+}
+
+export function test_decode_multiple_params(encoded_allocation: u32): u32 {
+  let jsonString: string = deserialize(encoded_allocation);   
```

Comment:
I got the impression from Max's PR we could/should just leave out the type defs on variable declarations that aren't in the function signature